go.mod

```diff
@@ -1,14 +1,14 @@
 module github.com/containerd/nerdctl/v2
 
-go 1.21
+go 1.21.12
```

Comment:
Can dependabot keep updating this?
Otherwise probably this should be .0?

Reply:
I do not think dependabot will up this.
I also prefer .0 (as I do not think we should be in the business of forcing people to the last patch update), but @fahedouch prefers to use the last patch (https://github.com/containerd/nerdctl/pull/3216#discussion_r1676892285)

@fahedouch what are your thoughts at this point?

Reply:
from a security perspective I think forcing people to use the last patch update is better. But since it's the supported version I am also ok for `.0`

Reply:
> Can dependabot keep updating this?

I don't think so

Reply:
Thanks a lot @fahedouch 

I honestly do not quite care either way - especially since 1.21 is soon to be out of support (and containerd v2 forces 1.22.0) - so, maybe we just leave it as-is (1.21.12) and revisit early August for go 1.23 release or our v2 release - whichever comes first.